services/graylog/docker-compose.yml

```diff
@@ -11,6 +11,13 @@ services:
       replicas: 1
       restart_policy:
         condition: on-failure
+      resources:
+        limits:
+          memory: 1.2G
+          cpus: "0.1"
```

Comment:
MongoDB with such low CPU, grayling will never reply in time. Also this one likes to have some ram so it replies faster to queries.

Reply:
Updated to "1"